docs/rules/mapStateToProps-no-store.md

```diff
@@ -1,7 +1,7 @@
 #  Enforces that mapStateToProps does not bind complete store to a component. (react-redux/mapStateToProps-no-store)
 
-Passing whole state to a component is a bd practice. Triggering unnecessary re-renders.
-Instead one should provide specific properties used by a component.
+Passing whole state to a component is a bad practice, triggering unnecessary re-renders.
```

Comment:
It’s also a bad practice because you’re passing around a mutable object that your component critically depends on preventing mutations to.

Reply:
Added.